models/apple_store__territory_report.sql

```diff
@@ -1,95 +1,98 @@
 with app as (
+    select
+        app_id,
+        app_name,
+        source_relation
+    from {{ var('app_store_app') }}
+),
 
+impressions_and_page_views as (
     select * 
-    from {{ var('app') }}
+    from {{ ref('int_apple_store__territory_impressions_page_views') }}
 ),
 
-app_store_territory as (
-
+downloads_daily as (
     select *
-    from {{ var('app_store_territory') }}
+    from {{ ref('int_apple_store__territory_downloads_daily') }}
 ),
 
-country_codes as (
-
-    select * 
-    from {{ var('apple_store_country_codes') }}
+install_deletions as (
+    select *
+    from {{ ref('int_apple_store__territory_install_deletions') }}
 ),
 
-downloads_territory as (
-
+sessions_activity as (
     select *
-    from {{ var('downloads_territory') }}
+    from {{ ref('int_apple_store__territory_sessions_activity') }}
 ),
 
-usage_territory as (
-
+country_codes as (
+    
     select * 
-    from {{ var('usage_territory') }}
+    from {{ var('apple_store_country_codes') }}
 ),
 
 reporting_grain as (
-
-    select distinct
-        source_relation,
-        date_day,
-        app_id,
-        source_type,
-        territory 
-    from app_store_territory
+    select *
+    from {{ ref('int_apple_store__territory_report') }}
 ),
 
-joined as (
-
-    select 
-        reporting_grain.source_relation,
-        reporting_grain.date_day,
-        reporting_grain.app_id,
-        app.app_name,
-        reporting_grain.source_type,
-        reporting_grain.territory as territory_long,
+-- Final aggregation using reporting grain
+final as (
+    select
+        rg.source_relation,
+        rg.date_day,
+        rg.app_id,
+        a.app_name,
+        rg.source_type,
+        rg.territory as territory_long,
```

Comment:
See note above if this is required
```suggestion
        coalesce(official_country_codes.country_name, alternative_country_codes.country_code_alpha_2) as territory_long,
```

Reply:
Since country_code_alpha_2  is short-form, I updated territory_long to now:

`        coalesce(country_codes.alternative_country_name,country_codes.country_name) as territory_long,`